aten/src/ATen/native/cuda/EmbeddingBag.cu

```diff
@@ -281,5 +250,143 @@ Tensor embedding_bag_backward_cuda(const Tensor &grad_, const Tensor &indices,
   THCudaCheck(cudaGetLastError());
   return grad_weight;
 }
+
+template <typename scalar_t>
+__global__ void EmbeddingBag_accGradParametersKernel_max(
+    int64_t *max_indices, scalar_t *gradOutput,
+    scalar_t *gradWeight, int64_t stride, int64_t numBags) {
+
+  using accscalar_t = cuda::acc_type<scalar_t>;
+
+  int64_t chunksPerBag = THCCeilDiv(stride, (int64_t)blockDim.x);
+  int64_t numChunks = numBags * chunksPerBag;
+  int64_t chunkOffset = blockIdx.x * blockDim.y + threadIdx.y;
+  int64_t chunkStride = gridDim.x * blockDim.y;
+
+  for (int64_t chunk = chunkOffset; chunk < numChunks; chunk += chunkStride) {
+    int64_t featureDim = (chunk % chunksPerBag) * blockDim.x + threadIdx.x;
+    if (featureDim < stride) {
+      int64_t bag = chunk / chunksPerBag;
+
+      int64_t word_idx = max_indices[bag * stride + featureDim];
+      
+      atomicAdd(&(gradWeight[word_idx * stride + featureDim]), gradOutput[bag * stride + featureDim]);
```

Comment:
out of scope of this PR but it'd be good to rewrite the kernel to avoid atomicAdd so that it becomes deterministic

Reply:
I'll look into this once and create a separate PR for that once this PR lands. I was mainly just copying the overall logic of SpatialAdaptiveMaxPooling.cu (which uses atomicAdd) for this initial PR. In theory it should be rather straightforward, just sort by the target location just like how the rest of the embedding bag code works.